cclib/parser/orcaparser.py

```diff
@@ -1612,6 +1627,17 @@ def parse_charge_section(self, line, inputfile, chargestype):
         elif chargestype == 'chelpg':
             should_stop = lambda x: x.startswith('---')
             start, stop = 11, 26
+        elif chargestype == 'hirshfeld':
+            # stops when blank line encountered
+            should_stop = lambda x: not bool(x.strip())
+            start, stop = 9, 17
+            for i in range(0, 3):
```

Comment:
'i' should vary from 0 to 2 rather than 0 to 3 to take all atoms into account.
The line should instead read: for i in range(0, 2):